Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidationTemplates.py

```diff
@@ -0,0 +1,374 @@
+####################################################################
+process.load("TrackingTools.TransientTrack.TransientTrackBuilder_cfi")
+
+####################################################################
+# Get the Magnetic Field
+####################################################################
+process.load("Configuration.StandardSequences..oO[magneticField]Oo._cff")
+
+###################################################################
+# Geometry load
+###################################################################
+process.load("Configuration.Geometry.GeometryRecoDB_cff")
+
+####################################################################
+# Get the BeamSpot
+####################################################################
+process.load("RecoVertex.BeamSpotProducer.BeamSpot_cff")
+
+####################################################################
+# Get the GlogalTag
+####################################################################
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '.oO[GlobalTag]Oo.', '')
+
+.oO[condLoad]Oo.
+     
+####################################################################
+# Load and Configure event selection
+####################################################################
+process.primaryVertexFilter = cms.EDFilter("VertexSelector",
+                                           src = cms.InputTag(".oO[VertexCollection]Oo."),
+                                           cut = cms.string("!isFake && ndof > 4 && abs(z) <= 24 && position.Rho <= 2"),
+                                           filter = cms.bool(True)
+                                           )
+
+process.noscraping = cms.EDFilter("FilterOutScraping",
+                                  applyfilter = cms.untracked.bool(True),
+                                  src =  cms.untracked.InputTag(".oO[TrackCollection]Oo."),
+                                  debugOn = cms.untracked.bool(False),
+                                  numtrack = cms.untracked.uint32(10),
+                                  thresh = cms.untracked.double(0.25)
+                                  )
+
+process.noslowpt = cms.EDFilter("FilterOutLowPt",
+                                applyfilter = cms.untracked.bool(True),
+                                src =  cms.untracked.InputTag(".oO[TrackCollection]Oo."),
+                                debugOn = cms.untracked.bool(False),
+                                numtrack = cms.untracked.uint32(0),
+                                thresh = cms.untracked.int32(1),
+                                ptmin  = cms.untracked.double(.oO[ptCut]Oo.),
+                                runControl = cms.untracked.bool(.oO[runControl]Oo.),
+                                runControlNumber = cms.untracked.vuint32(int(runboundary))
+                                )
```

Comment:
@mmusich one could load the default config here and just override the necessary parameters